react/features/shared-video/components/web/VideoManager.js

```diff
@@ -162,9 +160,7 @@ class VideoManager extends AbstractVideoManager<Props> {
             autoPlay: true,
             src: videoId,
             controls: _isOwner,
-            onError: event => {
-                logger.error('Error in the player:', event);
-            },
+            onError: () => this.onError(),
```

Comment:
this manager is for direct video links. The same behaviour should be added for the YoutubeVideoManager